#!/usr/bin/env python
PACKAGE = "minigame"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# add new parameters here
# gen.add("name", type, description, default, min, max)
gen.add("current_pos_x", double_t, 0, "current x-position",    0.0, 0,   100.0)
gen.add("current_pos_y", double_t, 0, "current y-position",    0.0, 0,   100.0)


exit(gen.generate(PACKAGE, "minigame", "point_a_node"))
